src/Management/src/EndpointCore/ManagementWebHostBuilderExtensions.cs

```diff
@@ -0,0 +1,278 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Steeltoe.Common.HealthChecks;
+using Steeltoe.Extensions.Logging;
+using Steeltoe.Management.Endpoint.CloudFoundry;
+using Steeltoe.Management.Endpoint.DbMigrations;
+using Steeltoe.Management.Endpoint.Env;
+using Steeltoe.Management.Endpoint.Health;
+using Steeltoe.Management.Endpoint.HeapDump;
+using Steeltoe.Management.Endpoint.Hypermedia;
+using Steeltoe.Management.Endpoint.Info;
+using Steeltoe.Management.Endpoint.Loggers;
+using Steeltoe.Management.Endpoint.Mappings;
+using Steeltoe.Management.Endpoint.Metrics;
+using Steeltoe.Management.Endpoint.Refresh;
+using Steeltoe.Management.Endpoint.ThreadDump;
+using Steeltoe.Management.Endpoint.Trace;
+using Steeltoe.Management.Info;
+using System;
+
+namespace Steeltoe.Management.Endpoint
+{
+    public static class ManagementWebHostBuilderExtensions
+    {
+        /// <summary>
+        /// Adds the Database Migrations actuator to the application
+        /// </summary>
+        /// <param name="hostBuilder">Your HostBuilder</param>
```

Comment:
Haven't looked at other comments ... "Your" seems informal.  Is that what similar comments in code base use?

Reply:
Contents of this file were copied from `ManagementHostBuilderExtensions` so it is done this way at least there, probably elsewhere as well... there's not a lot to say here... we could do `<see cref="IWebHostBuilder" />` instead

Reply:
Best left for future left then